app/root.tsx

```diff
@@ -36,6 +37,8 @@ export async function loader({ request, context }: LoaderArgs) {
   });
 }
 
+addReconcilerJob();
```

Comment:
I'll file an issue on this for later.  I don't like this pattern, and yet I have no good ideas to offer instead.

Reply:
Filed #450

Reply:
I agree, thank you!